File: qc-core/src/main/java/com/eager/questioncloud/core/domain/userquestion/dto/UserQuestionDetail.java
package com.eager.questioncloud.core.domain.userquestion.dto;

import com.eager.questioncloud.core.domain.question.enums.QuestionLevel;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class UserQuestionDetail {
    private Long questionId;
    private String title;
    private String parentCategory;
    private String childCategory;
    private String thumbnail;
    private String creatorName;
    private QuestionLevel questionLevel;
    private String fileUrl;
    private String explanationUrl;
}

File: qc-core/src/main/kotlin/com/eager/questioncloud/core/domain/userquestion/dto/UserQuestionDetail.kt
package com.eager.questioncloud.core.domain.userquestion.dto

import com.eager.questioncloud.core.domain.question.enums.QuestionLevel

class UserQuestionDetail(
    val questionId: Long,
    val title: String,
    val parentCategory: String,
    val childCategory: String,
    val thumbnail: String,
    val creatorName: String,
    val questionLevel: QuestionLevel,
    val fileUrl: String,
    val explanationUrl: String,
)
